### PR TITLE
Trying to the spacing of date range picker in advanced filtering dialog

### LINKED ISSUE #630: Advanced Filter: Date option in filter modal needs couple of visual fixes
## Description

Couple of discrepancies between the spec and Filter modal:

1. Spec has timestamps included in the default view of range picker. Unsure if that should be just a visual fix with range picker's attributes showing a timestamp or fixing it would actually include implementing a "Time" column in the grid(in addition to number, string, date types).
WE WILL REMOVE THIS:
2. Header Text should be changed to match the spec: 
- Current: "FROM DATE" -> Should be: "FROM" 
-  Current: "TO DATE" -> Should be: "TO"

## Version
Data-grid: V1.1.11

### Timestamps in Spec vs. Code

screen shot 2018-02-08 at 4 28 30 pm

### Header Text

screen shot 2018-02-08 at 5 19 02 pm

### PR DESCRIPTION
**Pull request by alump**
Fixes #630

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
     "px-actionable-design": "^2.0.5",
     "px-popover": "^2.1.2",
     "px-slider": "^2.1.1",
-    "px-rangepicker": "^3.2.0",
+    "px-rangepicker": "v3.3.4",
```
Comment (limonte) [inline bower.json]: why was `^` changed to `v`? That means we are stuck with the particular version and won't get any bug-fixes.

Comment (alump) [reply bower.json]: My mistake, fixing before merging. Good catch.

**Comment by alump** [comment]
There will be more changes done to px-slider, so this PR should wait until then.

**Comment by alump** [comment]
This version wasn't good, waiting new version of px-slider with CSS grid layouting.

**Pull request closed without merging**